policies/coco-operating-policies.md

```diff
@@ -0,0 +1,120 @@
+# Astropy Coordination Committee Operating Policies
+
+This living document outlines the guiding procedures of the Astropy Coordination
+Committee.  Note that this is not meant to be exhaustive nor duplicative - for
+example, the running notes doc documents how it is meant to be used itself.
+This document instead is meant to describe procedure, and where necessary,
+policy, for how the CoCo operates.
+
+Note also: [APE0](https://github.com/astropy/astropy-APEs/blob/main/APE0.rst)
+lays out the role, election, and philosophy for the CoCo, and that content is not
+duplicated here.
+
+## Meeting Details
+
+All meetings are held remotely (using Astropy’s Zoom) except when CoCo members happen to be co-located.
+
+*Frequency*
+
+Meeting frequency can vary depending on required workload with the assent of
+the current CoCo.  The minimum expected rate is monthly
```

Comment:
```suggestion
the current CoCo.  The minimum expected rate is monthly.
```
I wonder if we ever get there. The finance committee started weekly meetings in the expectation that we'd be able to scale back at some point. That was two years ago and we've made it through the agenda with no items left twice in weekly meetings.

Reply:
Are you suggesting a change? Useful context but I think it's fine to leave the minimum expected rate as monthly or biweekly. No strong feelings though..

Reply:
The change I suggest is adding "." at the end of the sentence. The test is just a comment.